cookbooks/aws-parallelcluster-config/files/default/imds/imds-access.sh

```diff
@@ -116,47 +120,54 @@ main() {
       shift
   done
 
-  # Check required commands
-  command -v iptables >/dev/null || error "Cannot find required command: iptables"
-
-  # Check arguments and options
-  if [[ -z $allow_users && -z $deny_users && -z $unset_users && -z $flush ]]; then
-    error "Missing at least one mandatory option: '--allow', '--deny', '--unset', '--flush'"
-  fi
-
-  # Setup ParallelCluster chain
-  setup_chain $PARALLELCLUSTER_CHAIN $OUTPUT_CHAIN $IMDS_IP
-
-  # Flush ParallelCluster chain, if required
-  if [[ $flush == "true" ]]; then
-    iptables --flush $PARALLELCLUSTER_CHAIN
-    info "ParallelCluster chain flushed"
-    exit 0
-  fi
-
-  # Delete rule: ACCEPT/REJECT user, for every user to unset
   IFS=","
-  for user in $unset_users; do
-    info "Deleting rules related to IMDS access for user: $user"
-    iptables_delete $PARALLELCLUSTER_CHAIN $IMDS_IP "ACCEPT" $user
-    iptables_delete $PARALLELCLUSTER_CHAIN $IMDS_IP "REJECT" $user
-  done
-
-  # Add rule: ACCEPT user, for every allowed user
-  for user in $allow_users; do
-    info "Allowing IMDS access for user: $user"
-    iptables_add $PARALLELCLUSTER_CHAIN $IMDS_IP "ACCEPT" $user
+  for ip_version_address in iptables,$IMDS_IP ip6tables,$IMDS_IP6
```

Comment:
Minor: I'ld put the 2 iptables command as constant
```bash
IPTABLE_CMD_V4="iptables"
IPTABLE_CMD_V6="ip6tables"
```

Reply:
Done